packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -163,6 +163,15 @@ const renderQueue = new RenderQueue(stack, 'RenderQueue', {
 });
 ```
 
+### Enabling Deadline Secrets Management on the Render Queue
+
+When [Deadline Secrets Management](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html) is enabled on the `Repository` construct,
+the `RenderQueue` will automatically configure itself as a Server role in Deadline Secrets Management.
```

Comment:
Can we link "Server role" to https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html#assigned-roles ?

Reply:
Sure, linked